tcms/settings/common.py

```diff
@@ -171,6 +172,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "simple_history.middleware.HistoryRequestMiddleware",
+    "django_prometheus.middleware.PrometheusAfterMiddleware",
```

Comment:
There are 2 independent items in this PR and IMO it's better to deal with them independently. 

Can you open a new PR which only contains the helm charts so I can review only that and work towards merging it.

Reply:
@atodorov I will do, just need your help then. Helm chart has livenessProbe and readinessProbe that rely upon any URL that returns 200 OK code. This is why I have introduced metrics. Can you tell me please which other existing URL returns 200 so I could use it?

Reply:
Any URL within Kiwi TCMS will return 200 when the container is up and running.  If you need a hard-coded 200 (e.g. can't deal with redirects) then `/accounts/login/` should work.

Reply:
thank you
done